packages/utils/src/stacktrace.ts

```diff
@@ -2,21 +2,24 @@ import { StackFrame } from '@sentry/types';
 
 const STACKTRACE_LIMIT = 50;
 
-export type StackLineParser = (line: string) => StackFrame | undefined;
+export type StackLineParserFn = (line: string) => StackFrame | undefined;
+export type StackLineParser = [number, StackLineParserFn];
 
 /**
  * Creates a stack parser with the supplied line parsers
  *
  * StackFrames are returned in the correct order for Sentry Exception
  * frames and with Sentry SDK internal frames removed from the top and bottom
  *
- * */
-export function createStackParser(...parsers: StackLineParser[]) {
+ */
+export function createStackParser(...parsers: StackLineParser[]): (stack: string, skipFirst?: number) => StackFrame[] {
```

Comment:
What do you think about extracting `(stack: string, skipFirst?: number) => StackFrame[]` into a type alias?